library/src/main/res/values-v9/strings.xml

```diff
@@ -2,5 +2,5 @@
 <!-- Copyright 2012 Square, Inc. -->
 
 <resources>
-  <string name="month_name_format">LLLL yyyy</string>
+  <string name="month_name_format" translatable="false">LLLL yyyy</string>
```

Comment:
Good point; i assumed this would be handled by the Java locale and this siliences some lint errors/warnings. I created #234 which only adds the translations.